apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py

```diff
@@ -1,56 +1,83 @@
 import time
 from datetime import datetime, timedelta
 
+from django.conf import settings
 from django.core.management.base import BaseCommand, CommandError, CommandParser
+from django.template.loader import render_to_string
 
 from apps.organization.models import Organization
 from apps.shared_revenue.services.split_export import SplitExportService
+from apps.shared_revenue.tasks import send_email_to_organization
 
 
 class Command(BaseCommand):
     """
-
     This command triggers the export of all the transactions splitted based on the given parameters per organization.
 
     Required parameters:
         - start_date: YYYY-MM-DD
         - end_date: YYYY-MM-DD
+        - send_file: true / false
 
     How to use:
 
-        python manage.py export_split_revenue_per_organizations  {start_date} {end_date}
+        python manage.py export_split_revenue_per_organizations  {start_date} {end_date} {send_file}
 
         Exemple:
 
-            python manage.py export_split_revenue_per_organizations 2023-12-01 2024-01-01
-
+            python manage.py export_split_revenue_per_organizations 2023-12-01 2024-01-01 true
```

Comment:
This command can be a bit more explicit.
`python manage.py export_split_revenue_per_organizations 2023-12-01 2024-01-01 --send-email`
As the intention is to send an email message to the organization.